libraries/AP_OSD/AP_OSD.cpp

```diff
@@ -214,6 +214,22 @@ const AP_Param::GroupInfo AP_OSD::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("_W_ACRVOLT", 31, AP_OSD, warn_avgcellrestvolt, 3.6f),
 
+#if AP_OSD_EXTENDED_LNK_STATS
```

Comment:
```suggestion
#if AP_OSD_EXTENDED_LNK_STATS_ENABLED
```

We'll make this change in a future PR....

Reply:
Do you mean you think this is a typo or you'd rather have the define renamed?